tests/fixtures/payload.py

```diff
@@ -155,6 +155,10 @@ def random_city():
     return {"name": name, "geo": geo_points[name]}
 
 
+def random_location():
```

Comment:
This one is unused.

I guess you wanted to replace all hardcoded location in the congruence test to increase the confidence

Reply:
Yes, I was trying to introduce random generated geopoints to replace the hard coded city location. However, this introduces inconsistency on local mode and remote mode as `lat`/`lon` are floating numbers. And python/rust have subtle difference in floating representation. I think it can be addressed. But given the current implementation, I choose to use the harde coded location since it is enough for test validation.
Removed this function as it is unnecessary.